dev-tex/rubber/rubber-1.6.0_alpha1.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+DISTUTILS_USE_SETUPTOOLS="no"
+
+inherit distutils-r1
+
+if [[ ${PV} == "9999" ]] || [[ -n "${EGIT_COMMIT_ID}" ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.com/latex-rubber/${PN}.git"
+else
+	UPSTREAM_PV=$(ver_rs 3 -)
+	S="${WORKDIR}/${PN}-${UPSTREAM_PV}"
+	SRC_URI="https://gitlab.com/latex-rubber/${PN}/-/archive/${UPSTREAM_PV}/${PN}-${UPSTREAM_PV}.tar.bz2"
+	KEYWORDS="~amd64 ~ppc ~x86"
+fi
+
+DESCRIPTION="A LaTeX wrapper for automatically building documents"
+HOMEPAGE="https://gitlab.com/latex-rubber/rubber"
+
+LICENSE="GPL-3+"
+SLOT="0"
+
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="virtual/latex-base"
+
+# Test dependencies:
+# - app-text/texlive-core for rubber's 'cweave' test
+# - dev-lang/R for rubber's 'knitr' test (requires knitr R library, currently disabled)
+# - dev-texlive/texlive-latexextra for rubber's 'combine' test (currently disabled)
+DEPEND="
+	${RDEPEND}
+	virtual/texi2dvi
+	test? (
+		app-text/texlive-core
+		dev-texlive/texlive-latexextra
+		media-gfx/asymptote
+		dev-tex/pythontex
+	)
+"
+
+python_install() {
+	insinto /usr/share/zsh/site-functions
```

Comment:
I wonder if it makes sense to install this per-Python implementation. Just once would work, right?

Reply:
Good catch, it's now in `src_install`.